probe-rs/src/probe/ftdi/ftdaye/error.rs

```diff
@@ -0,0 +1,150 @@
+use std::fmt;
```

Comment:
the way jaylink handles errors is a bit strange and quite different from the way the rest of the probe-rs codebase does it. For jaylink I copied it as-is hoping to clean it up later, but new code maybe should use the probe-rs way (basically using `thiserror`)

Reply:
Yeah you're right here but I needed a breather after getting this far :)